build.gradle

```diff
@@ -102,6 +102,10 @@ rat {
 }
 
 allprojects {
+    tasks.withType(JavaCompile).configureEach {
+        options.fork = true
+    }
+
```

Comment:
Minor: Could you please use 2 spaces for indentation instead of 4? Just for consistency with the rest of the file.

Reply:
Ok,  two spaces now